test/server.test.js

```diff
@@ -69,3 +69,16 @@ test('listen should reject string port', async (t) => {
     t.equal(error.code, 'ERR_SOCKET_BAD_PORT')
   }
 })
+
+test('listen should not start server if received abort signal', t => {
```

Comment:
AbortSignal on listen exists since node 15.6.0

```suggestion
test('listen should not start server if received abort signal', { skip: semver.lt(process.version, '15.6.0') }, t => {
```
Also add 

```js
const semver = require('semver')
```

on top of the test file.